docs/CONTRIBUTING.md

```diff
@@ -7,6 +7,11 @@ git clone git@github.com:kinvolk/lokomotive.git
 cd lokomotive
 ```
 
+## Requirements
```

Comment:
I think it would be good to add this in separate commit, so the additions are logically separated, which allows to reference and revert them independently, if needed in the future.